server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -1327,6 +1327,118 @@
+            }
+          },
+          {
+            "kind": "Deployment",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                ["settings", "spec", "template", "spec", "priorityClassName"]
+              ],
+              "description": "Deployments can specify a PriorityClass, ensuring Pods created by them inherit priority levels for scheduling."
+            }
+          },
+          {
+            "kind": "StatefulSet",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                ["settings", "spec", "template", "spec", "priorityClassName"]
+              ],
+              "description": "StatefulSets manage stateful applications. PriorityClass can be applied to StatefulSets' Pods to prioritize them during scheduling."
+            }
+          },
+          {
+            "kind": "DaemonSet",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                ["settings", "spec", "template", "spec", "priorityClassName"]
+              ],
+              "description": "DaemonSets ensure all nodes run a copy of a Pod. PriorityClass assigns importance, impacting scheduling decisions for DaemonSet-managed Pods."
+            }
+          },
+          {
+            "kind": "ReplicaSet",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                ["settings", "spec", "template", "spec", "priorityClassName"]
+              ],
+              "description": "ReplicaSets inherit PriorityClass from Pod templates, ensuring consistent priority levels for all Pods managed by the ReplicaSet."
+            }
+          },
+          {
+            "kind": "Job",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                ["settings", "spec", "template", "spec", "priorityClassName"]
+              ],
+              "description": "PriorityClass in Kubernetes influences Job scheduling, ensuring high-priority Jobs execute first, critical for resource allocation and workload management."
+            }
+          },
+          {
+            "kind": "CronJob",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatedRef": [
+                ["settings", "spec", "jobTemplate", "spec", "template", "spec", "priorityClassName"]
+              ],
+              "description": "CronJobs can use PriorityClass to prioritize scheduled job execution, ensuring critical tasks are handled with precedence in Kubernetes."
```

Comment:
CronJobs can or do use?

"handled with precedence in Kubernetes." or "ensuring CronJobs are scheduled for execution in accordance with the precedence specified in the associated PriorityClass."?